src/stylesheets/elements/_typography.scss

```diff
@@ -68,4 +68,353 @@
 
 .line-height-05 {
   line-height: 0.5rem;
+}
+
+
+
+
+// @mixin h3 {
+//   font-size: units(3);
+//   @include u-text('bold', 'base-dark');
+//   line-height: units(4);
+// }
+
+// @mixin h4 {
+//   font-size: units(2.5);
+//   @include u-text('semibold', 'base-dark');
+//   line-height: units(3);
+// }
+
+// @mixin h5 {
+//   font-size: units(2);
+//   @include u-text('bold', 'base-dark');
+//   line-height: units(2.5);
+// }
+
+// @mixin h6 {
+//   font-size: units(2);
+//   @include u-text('bold', 'base-dark');
+//   line-height: units(2.5);
+// }
+
+
+
+h1, h2, h3, h4, h5, h6 {
+
+  &.heading {
+
+    &--utility-nav {
+      @include u-font('sans', 'lg');
+      @include u-text('primary'); 
+    }
+    
+    &--action-menu {
+      @include u-font('sans', 'sm');
+      @include u-text('semibold', 'base-dark');
+      
+      &--link {
+        @include u-font('sans', 'xs');
+        @include u-text('semibold');
+        
+        &--visited{
+          @include u-font('sans', 'xs');
+          @include u-text('semibold', 'base-dark');
+        }
+      }
+      // @include u-margin-y(0);
+    }
+    &--footer-nav {
+      
+      &--group-label {
+        @include u-font('sans', 'md');
+        @include u-text('bold', 'base-dark');
+      }
+
+      &--link {
+        @include u-font('sans', 'xs');
+        @include u-text('normal');
+      }
+    }
+
+    &--section {
+      @include u-font('sans', 'lg');
+      @include u-text('semibold', 'base-dark');
+
+      &--sub {
+        @include u-font('sans', 'md');
+        @include u-text('light');
+      }
+    }
+
+    &--modal {
+      @include u-font('sans', 'lg');
+      @include u-text('semibold', 'base-dark');
+    }
+  }
+}
+p {  
+  @include u-font('sans', 'xs');
+  @include u-text('normal', 'base-dark');
+  @include u-height(3);
+  @include u-measure(5);
+}
+
+.p {
+  @extend p;
+  
+  &--error{
+    @include u-text('error-dark');
+  }
+
+  &--large {
+    @include u-font('sans', 'md');
+    @include u-height(4);
+  }
+
+  &--small {
+    @include u-font('sans', '3xs');
+    @include u-height(2);
+  }
+
+  &.global{
+    @include u-font('sans', 'md');
+
+    &.title {
```

Comment:
These don't seem right either:


```
<p class="global title">title</p>

<p class="global text">text</p>

```

Reply:
These might be better nested under sds-callout


  &.sds-callout {
    &--title {}
    &--text {}
  }


```
<p class="sds-callout sds-callout--title">Title</p>
<p class="sds-callout sds-callout--text">Text</p>
```